.github/workflows/ci.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           - target: iOS
             host: macos-13
           - target: Linux
-            host: ubuntu-latest
+            host: ubuntu-20.04
```

Comment:
If we're pinning versions we should put a comment there why we do it. So it doesn't get undone by accident